packages/Search/index.js

```diff
@@ -90,10 +90,13 @@ export const Search = forwardRef(
       setResults([])
     }
 
+    // prevents controlled to uncontrolled switch when itemToString returns falsy value
+    const handleItemToString = item => itemToString(item) || ''
```

Comment:
```suggestion
    const handleItemToString = item => itemToString(item) ?? ''
```

Pour prevenir les items ou `itemToString(item)` === 0